shared-bindings/displayio/Display.c

```diff
@@ -246,6 +246,30 @@ STATIC mp_obj_t displayio_display_obj_set_auto_brightness(mp_obj_t self_in, mp_o
 }
 MP_DEFINE_CONST_FUN_OBJ_2(displayio_display_set_auto_brightness_obj, displayio_display_obj_set_auto_brightness);
 
+//|   .. attribute:: width
+//|
+//|	Gets the width of the board
+//|
+//|
+STATIC mp_obj_t displayio_display_obj_get_width(mp_obj_t self_in) {
+    displayio_display_obj_t *self = MP_OBJ_TO_PTR(self_in);
+    mp_int_t width = common_hal_displayio_display_get_width(self);
+    return mp_obj_new_int(width);
+}
+MP_DEFINE_CONST_FUN_OBJ_1(displayio_display_get_width_obj, displayio_display_obj_get_width);
```

Comment:
To make this a property in CP, you'll need to add an `mp_obj_property_t` like `displayio_display_width_obj` and add it to the locals dict instead of `displayio_display_get_width_obj`

This will allow for referencing like so:
`board.DISPLAY.width` or 
`board.DISPLAY.height`

See this PR for an example:
https://github.com/adafruit/circuitpython/pull/1479/files